bitnami/contour/templates/contour/ingressclass.yaml

```
@@ -0,0 +1,16 @@
+{{ $ingressClass := .Values.contour.ingressClass }}
+{{- if (kindIs "map" $ingressClass) }}
```

Are the parenthesis needed?
```suggestion
{{- if kindIs "map" $ingressClass }}
```